README.md

```diff
@@ -83,7 +83,50 @@ The CKAN core and base images can be built on Github actions by selecting the `B
 
 ## Deploying CKAN
 
-By default, merging a PR into main will trigger a build of the CKAN image and create PRs on the govuk-dgu-charts repo. Updating the tag will build and push a tagged image and create the PR for Staging and Production environments in the govuk-dgu-charts repo.
+### Environments and URLs
+
+- **Integration**: [https://www.integration.data.gov.uk/](https://www.integration.data.gov.uk/) (VPN required)
+- **Staging**: [https://www.staging.data.gov.uk/](https://www.staging.data.gov.uk/) (VPN required)
+- **Production**: [https://www.data.gov.uk/](https://www.data.gov.uk/) (public)
```

Comment:
`ckanext-datagovuk` is the publishing app: https://ckan.publishing.service.gov.uk 
`datagovuk_find` is the frontend/rendering public facing app: https://www.data.gov.uk

Looks like VPN is not required for CKAN
```suggestion
- **Integration**: [https://ckan.integration.publishing.service.gov.uk](https://ckan.integration.publishing.service.gov.uk/)
- **Staging**: [https://ckan.staging.publishing.service.gov.uk](https://ckan.staging.publishing.service.gov.uk//)
- **Production**: [https://ckan.publishing.service.gov.uk/](https://ckan.publishing.service.gov.uk/)
```